runtime/include/localeModels/gpu/chpl-locale-model.h

```diff
@@ -42,6 +42,8 @@ typedef struct {
 //
 #define CHPL_LOCALEID_T_INIT  {0, 0}
 
+#define INITIAL_SUBLOCALE -1
```

Comment:
do we use this anywhere?

Reply:
No, I think I added this back when we were debugging this and wanted to ensure that we started new tasks on the "none" rather than the "any" sublocale. It looks like I reverted the piece of code where we used this but forgot to remove this `#define` itself. I'll remove this `#define` for the time being but I might end up reintroducing later if we do end up needing to worry about what sublocale tasks start on.